Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/SiteSettings/LanguagesControllerTasks.cs

```diff
@@ -131,7 +131,7 @@ public static void LocalizeLanguagePages(LocalizationProgress progress, int port
         internal static LocalizationProgress ReadProgressFile()
         {
             var path = Path.Combine(Globals.ApplicationMapPath, "App_Data", LocalizationProgressFile);
-#if true
+#if false
```

Comment:
This is confusing to me, it`s a compiler directive on a constant value?

Reply:
Yeah, it's like someone commenting out code. I guess I could also just kill the old code. Thoughts?

Reply:
I just removed the dead code.

Reply:
Sounds good to me